doc/source/development/contributing.rst

```diff
@@ -325,7 +325,10 @@ Creating a Python environment (pip)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you aren't using conda for your development environment, follow these instructions.
-You'll need to have at least Python 3.6.1 installed on your system.
+You'll need to have at least Python 3.6.1 installed on your system. If your Python version
```

Comment:
3.7 now?

Reply:
Thanks! Updated!

Reply:
my bad, just noticed there's already #40552 open to address the 3.6.1 issue, sorry for my irrelevant comment